html/global_attributes.json

```
@@ -1016,9 +1016,22 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": false
-            },
+            "firefox": [
+              {
+                "version_added": "17",
+                "notes": "Prior to 75, Firefox used values from an earlier spec version; in 75, it was updated to match the WHATWG Living Standard as of that time. However, Firefox does not support inputmode when contenteditable is true (see https://bugzil.la/1424284)."
```

I think these might be two distinct notes, like this:

```suggestion
                "notes": [
                  "Firefox does not support <code>inputmode</code> when <code>contenteditable</code> is <code>true</code> (see <a href='https://bugzil.la/1424284'>bug 1424284</a>)."
                  "Prior to 75, Firefox used values from an earlier spec version; in 75, it was updated to match the WHATWG Living Standard as of that time."
                ]
```

(Also a little bit of formatting added)